src/notifications/templates/notifications/email/subscription_cancellation_confirmed.html

```
@@ -0,0 +1,38 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        body {margin:0;padding:0;font-family:-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,'Helvetica Neue',Arial,sans-serif;}
+        .container {max-width:600px;margin:0 auto;background:white;}
+        .header {background:linear-gradient(135deg,#6c757d 0%,#495057 100%);color:white;padding:30px 20px;text-align:center;}
+        .content {padding:30px 20px;}
+        .card {background:#f8f9fa;border-left:4px solid #6c757d;padding:20px;margin:20px 0;border-radius:4px;}
+        .info-row {margin:10px 0;}
+        .label {color:#666;font-weight:600;}
+        .footer {background:#f8f9fa;padding:20px;text-align:center;color:#666;font-size:14px;}
+    </style>
+</head>
```

HTMLHint (1.9.2)

[error] <title> must be present in <head> tag.

(title-require)